dvc/command/pipeline.py

```diff
@@ -8,15 +8,26 @@
 logger = logging.getLogger(__name__)
 
 
+def _stage_repr(stage):
```

Comment:
Why not just bake it into stage objects? :slightly_smiling_face:

Reply:
Will come to this when I fix UI issues. The reason why I haven't fixed this is because, I want to make `__repr__` to be `Stage: <abspath>:<name>` and `__str__` to be `Stage: <relpath>:<name>`.  It feels like this should be `get_address` or `get_address_name` or something. Anyway, I'll revisit it later on.